nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/state/shared/index.ts

```diff
@@ -120,6 +120,11 @@ export interface UpdateReportingTaskRequest {
     postUpdateNavigation?: string[];
 }
 
+export interface UpdateFlowAnalysisRuleRequest {
```

Comment:
This interface definition (and the `UpdateReportingTaskRequest` above) could be moved to their respective `pages`. `UpdateConfigurationServiceRequest` is here because the Controller Service edit dialog is a shared component between the Canvas and Controller Services. The Flow Analysis edit dialog and Reporting Task edit dialog are not shared components.